lldb/include/lldb/API/SBDefines.h

```diff
@@ -34,7 +34,9 @@
 // supports the attribute.
 #if defined(SWIG) or _cplusplus < 201402L
```

Comment:
Can https://github.com/llvm/llvm-project/commit/984dd163a7a03c517e419475899b80dda841e752 be cherry-picked on top of these changes? Otherwise Swig will not understand the macros in the other files.